platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/diagram-commands.txt

```diff
@@ -28,20 +27,25 @@ Same as 'Uncollapsed' but with low level things collapsed. Attempts to hide thin
 pcli diagram \
     -l 'applicationTransactionPrehandler:futures:linkedEventIntake' \
     -s 'getNonAncientEventWindow:non-ancient event window:ʘ' \
-    -s 'getMinimumGenerationNonAncient:minimum generation non ancient:ẞ' \
     -s 'heartbeat:heartbeat:♡' \
     -s 'eventCreationManager:non-validated events:†' \
     -s 'applicationTransactionPrehandler:futures:★' \
     -l 'eventDurabilityNexus:countUpLatch:linkedEventIntake' \
     -g 'Event Validation:internalEventValidator,eventDeduplicator,eventSignatureValidator' \
     -g 'Orphan Buffer:orphanBuffer,orphanBufferSplitter' \
-    -g 'Linked Event Intake:linkedEventIntake,linkedEventIntakeSplitter,getNonAncientEventWindow,getMinimumGenerationNonAncient' \
+    -g 'Linked Event Intake:linkedEventIntake,linkedEventIntakeSplitter,getNonAncientEventWindow' \
     -g 'State File Management:signedStateFileManager,extractOldestMinimumGenerationOnDisk,toStateWrittenToDiskAction' \
     -g 'State Signature Collection:systemTransactionsFilter,systemTransactionsSplitter,stateSignatureTransactionsFilter,stateSignatureCollector' \
     -g 'Intake Pipeline:Event Validation,Orphan Buffer,eventHasher' \
-    -g 'PCES:pcesSequencer,pcesWriter,eventDurabilityNexus' \
+    -g 'Preconsensus Event Stream:pcesSequencer,pcesWriter,eventDurabilityNexus' \
     -g 'Consensus Pipeline:inOrderLinker,Linked Event Intake' \
     -c 'Orphan Buffer' \
     -c 'Linked Event Intake' \
     -c 'State File Management' \
-    -c 'State Signature Collection'
+    -c 'State Signature Collection' \
+    -c 'Preconsensus Event Stream'
+
+
+
+
+        -s 'pcesReplayer:done streaming pces:✓' \
```

Comment:
This is clutter I introduced at some point, and eventually cleaned up. Must have made it back in via a merge

Reply:
Removed.